core/blockchain/impl/block_tree_impl.cpp

```diff
@@ -52,10 +52,12 @@ namespace kagome::blockchain {
           if (res.has_error()) {
             if (res
                 == outcome::failure(
-                    blockchain::BlockTreeError::HEADER_NOT_FOUND)) {
+                    blockchain::BlockTreeError::EXISTING_BLOCK_NOT_FOUND)) {
```

Comment:
Without any doubt - "existing" and "not found" at the same time sounds pretty confusing :)